### ISSUE TITLE
Can no longer start Nextcloud AIO

### ISSUE THREAD

**Issue by rderooy** [nextcloud/all-in-one#4056]
### Steps to reproduce
1. Start docker containers
2. fails to completely start
3.

### Expected behavior

all containers starting

### Actual behavior

![image](https://github.com/nextcloud/all-in-one/assets/17701098/7995f7ce-67cf-4491-bbaf-cad0d685a4d1)

```
NOTICE: PHP message: Slim Application Error
Type: Exception
Code: 0
Message: Could not pull image nextcloud/aio-nextcloud:latest. Please run &#039;sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest&#039; in order to find out why it failed.
```

```bash
$ sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest
latest: Pulling from nextcloud/aio-nextcloud
Digest: sha256:f22bc97311c2a4a51877d116a8a2a6fb8a0eb0abec1bee445755e3db9822dd35
Status: Image is up to date for nextcloud/aio-nextcloud:latest
docker.io/nextcloud/aio-nextcloud:latest
```

Stopping and starting the docker containers at this stage makes no difference.

### Host OS 

OpenMediaVault 6.9.11-4 (Debian 11) running on x86_64

#### Nextcloud AIO version

7.9.1

#### Current channel

latest

#### Other valuable info
```
Trying to fix docker.sock permissions internally...
Creating docker group internally with id 992
Initial startup of Nextcloud All-in-One complete!
You should be able to open the Nextcloud AIO Interface now on port 8080 of this server!
E.g. https://internal.ip.of.this.server:8080

If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
https://your-domain-that-points-to-this-server.tld:8443
++ head -1 /mnt/docker-aio-config/data/daily_backup_time
+ BACKUP_TIME=04:00
+ export BACKUP_TIME
+ export DAILY_BACKUP=1
+ DAILY_BACKUP=1
++ sed -n 2p /mnt/docker-aio-config/data/daily_backup_time
+ '[' '' '!=' automaticUpdatesAreNotEnabled ']'
+ export AUTOMATIC_UPDATES=1
+ AUTOMATIC_UPDATES=1
++ sed -n 3p /mnt/docker-aio-config/data/daily_backup_time
+ '[' successNotificationsAreNotEnabled '!=' successNotificationsAreNotEnabled ']'
+ export SEND_SUCCESS_NOTIFICATIONS=0
+ SEND_SUCCESS_NOTIFICATIONS=0
+ set +x
{"level":"info","ts":1705149453.3423848,"msg":"using provided configuration","config_file":"/Caddyfile","config_adapter":""}
[13-Jan-2024 12:37:33] NOTICE: fpm is running, pid 144
[13-Jan-2024 12:37:33] NOTICE: ready to handle connections
[Sat Jan 13 12:37:33.408031 2024] [mpm_event:notice] [pid 138:tid 140610504399688] AH00489: Apache/2.4.58 (Unix) OpenSSL/3.1.4 configured -- resuming normal operations
[Sat Jan 13 12:37:33.408072 2024] [core:notice] [pid 138:tid 140610504399688] AH00094: Command line: 'httpd -D FOREGROUND'
NOTICE: PHP message: Slim Application Error
Type: Exception
Code: 0
Message: Could not pull image nextcloud/aio-nextcloud:latest. Please run &#039;sudo docker exec -it nextcloud-aio-mastercontainer docker pull nextcloud/aio-nextcloud:latest&#039; in order to find out why it failed.
File: /var/www/docker-aio/php/src/Docker/DockerActionManager.php
Line: 590
Trace: #0 /var/www/docker-aio/php/src/Controller/DockerController.php(54): AIO\Docker\DockerActionManager->PullImage(Object(AIO\Container\Container))
#1 /var/www/docker-aio/php/src/Controller/DockerController.php(33): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-n...', true)
#2 /var/www/docker-aio/php/src/Controller/DockerController.php(202): AIO\Controller\DockerController->PerformRecursiveContainerStart('nextcloud-aio-a...', true)
#3 /var/www/docker-aio/php/src/Controller/DockerController.php(182): AIO\Controller\DockerController->startTopContainer(true)
#4 /var/www/docker-aio/php/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponse.php(38): AIO\Controller\DockerController->StartContainer(Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#5 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(358): Slim\Handlers\Strategies\RequestResponse->__invoke(Array, Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#6 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\Routing\Route->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#7 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#8 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(315): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#9 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(68): Slim\Routing\Route->run(Object(GuzzleHttp\Psr7\ServerRequest))
#10 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(476): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#11 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
#12 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#13 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#14 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#15 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#16 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#17 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#18 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#19 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#20 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#21 /var/www/docker-aio/php/public/index.php(185): Slim\App->run()
#22 {main}
```

**Comment by szaimen** [comment]
Hi, I fear I cannot reproduce this but will still try to imporve this going forward with https://github.com/nextcloud/all-in-one/pull/4066

**Comment by szaimen** [comment]
This is now released with v7.11.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Comment by rderooy** [comment]
Thank you, I can confirm that the issue is fixed.
